tools/pandapower_eval.py

```diff
@@ -0,0 +1,36 @@
+# In order to install PandaPower run 'pip install matplotlib pandapower'
+import pandapower as pp
+from pandapower.plotting import to_html
+
+
+# create empty net
+net = pp.create_empty_network()
+
+# create buses
+bus1 = pp.create_bus(net, vn_kv=20., name="Bus Grid 1")
+bus_trafo = pp.create_bus(net, vn_kv=0.4, name="Bus Trafo")
+bus_h1 = pp.create_bus(net, vn_kv=0.4, name="Bus House 1")
+bus_h2 = pp.create_bus(net, vn_kv=0.4, name="Bus House 2")
+
+# create bus elements
+pp.create_ext_grid(net, bus=bus1, vm_pu=1.0, name="Grid Connection")
+pp.create_load(net, bus=bus_h1, p_kw=0.1, q_kvar=0.005, name="H1 Load")
+pp.create_sgen(net, bus=bus_h1, p_kw=-0.1, name="H1 PV")
```

Comment:
In case of PV, we have power profile rather than one power value. So do we have to update p_kw for every market_slot?

Reply:
I think it was mentioned in the documentation too, none of these tools support dynamic analysis, only steady state analysis is supported for both pandapower and PyPSA. Therefore there is no straightforward way of providing 'power profiles' to the software and have the analysis ready (there is a way via 'snapshots', but it is the same as changing the values and manually performing the power flow analysis sequentially).

Reply:
Sorry if I missed it, for PyPSA it is explicitly written that it supports steady-state analysis but for pandapower I couldn't find it there. Thanks for the clarification